src/proper_target.erl

```diff
@@ -224,7 +257,7 @@ reset() ->
 safe_call(Pid, Call) ->
```

Comment:
Just saw this, what is this for? calling a gen_server that crashed returns ok?

Reply:
The reason this exists, is because at the moment the `?FORALL_TARGETED` and `?EXISTS`-`?NOT_EXISTS` macros work a bit different.
exv
**`?FORALL_TARGETED`**

Essentially, this is a wrapper for a `?FORALL` macro, which also sets up the `proper_target` `gen_server`. That means that the `gen_server` is "up" on all stages of testing (run and shrink).

**`?EXISTS` - `?NOT_EXISTS`**

These use the custom `perform_search` function, however the `proper_target` `gen_server` is only "up" during that phase. When shrinking the `gen_server` is "down", but the property keeps making calls to it through the `?MAXIMIZE`-`?MINIMIZE` macros. These are irrelevant during shrinking however, so a non existent `gen_server` should not raise any exception.

Reply:
It seems like this can lead to some relatively hard to detect bugs. E.g. the gen_server crashed somehow and all calls seem to succeed. 

In this case the system should loudly crash. Is there not the possibility(in another PR) to have the mode/phase forwarded to the API and ignore them there if we are shrinking?

Reply:
Yes, this is something that has been discussed.